ciao-image/datastore/posix.go

```diff
@@ -44,5 +44,12 @@ func (p *Posix) Write(ID string, body io.Reader) (int64, error) {
 func (p *Posix) Delete(ID string) error {
 	imageName := path.Join(p.MountPoint, ID)
 
-	return os.Remove(imageName)
+	_, err := os.Stat(imageName)
+	if !os.IsNotExist(err) && err != nil {
+		return err
+	}
+
+	os.Remove(imageName)
```

Comment:
What about checking the error from os.Remove?  if you want to ignore it, fair enough but assign it to _ to show that you really want to do this and that this is not just an oversight.